doc/build-instructions/Build_Instructions_V13.md

```diff
@@ -0,0 +1,651 @@
+               --with-freemarker-jar=/cygdrive/c/temp/freemarker.jar
+```
+
+
+:pencil: Modify the paths for freemarker if you manually downloaded and unpacked these dependencies into different directories.
+
+:pencil: **Non-compressed references support:** If you require a heap size greater than 57GB, enable a noncompressedrefs build with the `--with-noncompressedrefs` option during this step.
+
+:pencil: **OpenSSL support:** If you want to build an OpenJDK that includes OpenSSL, you must specify `--with-openssl=path_to_library`, where `path_to_library` specifies the path to the prebuilt OpenSSL library that you obtained in **2. Get the source**. If you want to include the OpenSSL cryptographic library in the OpenJDK binary, you must also include `--enable-openssl-bundling`.
+
+### 4. build
+:ledger:
+Now you're ready to build OpenJDK with OpenJ9:
+```
+make all
+```
+
+A binary for the full developer kit (jdk) is built and stored in the following directory:
+
+- **build/windows-x86_64-normal-server-release/images/jdk**
+
+    :pencil: If you want a binary for the runtime environment (jre), you must run `make legacy-jre-image`, which produces a jre build in the **build/windows-x86_64-normal-server-release/images/jre** directory.
+
+### 5. Test
+:ledger:
+For a simple test, try running the `java -version` command.
+Change to the /jdk directory:
+```
+cd build/windows-x86_64-normal-server-release/images/jdk
+```
+Run:
+```
+./bin/java -version
+```
+
+Here is some sample output:
+
+```
+openjdk version "13-internal" 2019-09-19
+OpenJDK Runtime Environment (build 13-internal+0-adhoc.jenkins.Build-JDK13-winx86-64cmprssptrs)
+Eclipse OpenJ9 VM (build tye-e85051733, JRE 13 Windows Server 2016 amd64-64-Bit Compressed References 20190906_27 (JIT enabled, AOT enabled)
+OpenJ9   - e85051733
+OMR      - dfbca14c
+JCL      - 2ef6b4c54d8 based on jdk-13+30)
+```
+
+:pencil: **OpenSSL support:** If you built an OpenJDK with OpenJ9 that includes OpenSSL support, the following acknowledgements apply in accordance with the license terms:
+
+  - *This product includes software developed by the OpenSSL Project for use in the OpenSSL Toolkit. (http://www.openssl.org/).*
+  - *This product includes cryptographic software written by Eric Young (eay@cryptsoft.com).*
+
+:ledger: *Congratulations!* :tada:
+
+----------------------------------
+
+## macOS
+:apple:
+The following instructions guide you through the process of building a macOS **OpenJDK V13** binary that contains Eclipse OpenJ9. This process can be used to build binaries for macOS 10.
+
+### 1. Prepare your system
+:apple:
+You must install a number of software dependencies to create a suitable build environment on your system:
+
+- [Xcode 9.4]( https://developer.apple.com/download/more/) (requires an Apple account to log in).
+- [macOS OpenJDK 12](https://adoptopenjdk.net/archive.html?variant=openjdk12&jvmVariant=hotspot), which is used as the boot JDK.
```

Comment:
OpenJ9 is preferred over Hotspot.

Reply:
Yep ... must be from the days when we didn't have one to use. Fixed.